src/RoomInfoListItem.cpp

```diff
@@ -324,6 +329,34 @@ RoomInfoListItem::updateUnreadMessageCount(int count, int highlightedCount)
         update();
 }
 
+enum NotificationImportance : short
+{
+        ImportanceDisabled = -1,
+        AllEventsRead      = 0,
+        NewMessage         = 1,
+        NewMentions        = 2,
+        Invite             = 3
+};
+
+unsigned short int
```

Comment:
If you are returning a short, you should probably not convert it to unsigned here, since that leads to Invites sticking to the bottom instead of the top

Reply:
Oop good catch, sorry abt that

Reply:
No worries, it happens. Just remember to actually check all places, that use unsigned ;-)